pkg/controller/build/build_controller.go

```diff
@@ -125,6 +131,9 @@ type BuildControllerConfig struct {
 	// 5ms, 10ms, 20ms, 40ms, 80ms, 160ms, 320ms, 640ms, 1.3s, 2.6s, 5.1s, 10.2s, 20.4s, 41s, 82s
 	// Default: 5
 	MaxRetries int
+
+	// PreviousConfig is used in the case of an interrupted build. This ensures that if a build didn't finish or it didn't even start, we can keep using the old rendered config
+	PreviousConfig string
```

Comment:
question: Is this being used anywhere?

Reply:
no, I'll delete this. I was leaving it as a mental marker. But sine `.Source` is an array of configs, I can just remove the last entry of the interrupted build